src/service/url-expander/expander.js

```diff
@@ -142,14 +142,18 @@ export class Expander {
 
     const evaluateNextLevel = encode => {
       let builder = '';
-      const results = [];
+      let results = [];
+      const args = [];
 
       while (urlIndex < url.length && matchIndex <= matches.length) {
         if (match && urlIndex === match.start) {
```

Comment:
Have a question about the usage of backtick. 
What's the expected output with
```
abc`TIMESTAMP`def
```
I couldn't find where we respect the `PARSER_IGNORE_FLAG` when handling with macros. Do you think that's an issue?

Reply:
You are right this macro is still resolved. The backtick only really works for spaces, parens, and commas. I think this is ok as we note in docs that macro names are reserved, but happy to revisit this. If we were to change it would need to be a separate PR.

Reply:
> I think this is ok as we note in docs that macro names are reserved

As long as we've made that decision, I think it makes sense. could you please remind me when did we add support for backtick (before nested macro support, or after).

Reply:
After.

Reply:
Cool. I think reserve macros makes sense because I believe we have no plan to support keeping the `${vars}` anyway.